### PR TITLE
Resume link

### PR DESCRIPTION
**Pull request by vdvibhu20**
### Is this PR regarding CSS? 
This project uses motley(https://github.com/coding-blocks/motley/) as css framework. We don't accept css changes here. Please consider a PR to motley instead

### CODE REVIEW

File: lib/hiring-blocks/addon/pods/listing/job/template.hbs [hunk 1]
```diff
@@ -131,4 +131,18 @@
 </section>
 
 <PartnersSection />
+{{#if showModal}}
+{{#modal-dialog
+  onClose=(action toggleShowModal)
```
Comment (abhishek97) [inline lib/hiring-blocks/addon/pods/listing/job/template.hbs]: Use Angle Bracket Invocation for better readability

File: lib/hiring-blocks/addon/pods/listing/job/controller.js [hunk 2]
```diff
@@ -18,16 +20,21 @@ export default class ListingJobController extends Controller {
...
   @restartableTask
   *applyForJob () {
-    console.log("here")
     yield this.store.createRecord('application', {
-      job: this.job
+      job: this.job,
+      resumeLink: this.resumeLink
     }).save()
-
+    .then(() => this.toggleShowModal())
```
Comment (abhishek97) [inline lib/hiring-blocks/addon/pods/listing/job/controller.js]: Since you've already yielded the promise, `.then()` is not required here.

**Comment by vdvibhu20** [comment]
@abhishek97 Done

File: lib/hiring-blocks/addon/pods/listing/job/controller.js [hunk 3]
```diff
@@ -18,16 +20,21 @@ export default class ListingJobController extends Controller {
 
   @restartableTask
```
Comment (abhishek97) [inline lib/hiring-blocks/addon/pods/listing/job/controller.js]: use `dropTask`

**Pull request merged**